test/parallel/test-http2-compat-write-early-hints.js

```diff
@@ -128,7 +128,9 @@ const testResBody = 'response content';
 
     debug('Client sending request...');
 
-    req.on('headers', common.mustNotCall());
+    req.on('headers', common.mustCall((headers) => {
+      assert.strictEqual(headers[':status'], 103);
```

Comment:
This seems like a breaking change?

Reply:
Per my comment above about removing the check for the presence of the Link header I think it's a desirable change. But I see how it could be considered breaking. What are the implications of it being a breaking change?

Reply:
early hints support is still experimental, I think we are good.

Reply:
Although it is permitted to send a empty `103 Early Hints`. It is strange to send a `103 Early Hints` if there is no additional header.
Because it waste bandwidth with no gain.